composer.lock

```diff
@@ -282,21 +282,72 @@
             },
             "time": "2020-04-20T17:47:06+00:00"
         },
+        {
```

Comment:
No changes to `composer.lock` were required so please undo them to avoid any potential issues later.